app/configs/locationDataCard.ts

```diff
@@ -0,0 +1,3 @@
+import { locationColumns } from "./locationColumns";
+
+export const locationCardColumns = [locationColumns.location_id];
```

Comment:
this feels a bit silly but i'm matching the pattern i used to set up the crash data card column arrays 🤷